scripts/azure-ci-test.sh

```diff
@@ -91,23 +98,22 @@ save_logs() {
 }
 
 cleanup() {
-  save_logs
```

Comment:
Should this be kept so we can upload the Azure test logs?

Reply:
right, it should be good after rebased.